### PR TITLE
Add more metrics tracking for new billing

### PR DESCRIPTION
**Pull request by AndriiMysko**

### CODE REVIEW

**Comment by AndriiMysko** [comment]
Hello @artursmirnov !
Just a small addition of more metrics tracking for the new billing system, please take a look.

File: app/components/billing/payment.js [hunk 1]
```diff
@@ -78,13 +78,26 @@ export default Component.extend({
 
   updatePlan: task(function* () {
     if (this.selectedPlan.isFree) {
+      this.metrics.trackEvent({
+        action: 'Plan Reverted to Free Plan',
+        category: 'Subscription',
+      });
```
Comment (artursmirnov) [inline app/components/billing/payment.js:84]: Can users cancel in the modal that this action opens? If yes, then this data may become inaccurate.

Comment (AndriiMysko) [reply app/components/billing/payment.js:84]: You're right, thank you!

**Review by artursmirnov — APPROVED** [review]
It looks great to me, thank you @AndriiMysko 👍

**Pull request merged**